pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -239,12 +245,136 @@ func (adc *attachDetachController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	err := adc.populateActualStateOfWorld()
```

Comment:
There is `controller.WaitForCacheSync()` just few lines above. Does it mean that all the existing objects were already synced through podAdd() and nodeAdd() callbacks from the shared informers? They may be synced in a wrong order (podAdd before nodeAdd) and these `populateActualStateOfWorld()` and `populateDesiredStateOfWorld()` will just fix it, right?